ruby/src/api.cc

```diff
@@ -86,7 +86,7 @@ namespace leatherman { namespace ruby {
         LOAD_SYMBOL(rb_to_id),
         LOAD_SYMBOL(rb_id2name),
         LOAD_SYMBOL(rb_define_alloc_func),
-        LOAD_SYMBOL(rb_data_object_alloc),
+        LOAD_ALIASED_SYMBOL(rb_data_object_alloc, rb_data_object_wrap),
```

Comment:
I wonder why they replaced `rb_data_object_alloc` with a method of the same signature?

For ref, Ruby deprecation in 2.3.0 appears at:
https://github.com/ruby/ruby/blob/v2_3_0/include/ruby/ruby.h#L1418-L1425

👍